src/Compilers/Core/Portable/Serialization/ObjectWriter.cs

```diff
@@ -57,18 +57,8 @@ internal sealed partial class ObjectWriter : IDisposable
         /// and not the objects themselves because the ID is inferred from the order the object is
         /// written or read in.
         /// </summary>
-        private WriterReferenceMap _objectReferenceMap;
         private WriterReferenceMap _stringReferenceMap;
```

Comment:
we still have the string refernece map.  this is still a useful part of serialization where if you write the same string many times into the stream (common with things like Ids, which often refer to the same project/document over and over again) that you just get a back-reference number.